specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json

```diff
@@ -1,7 +1,7 @@
 {
   "parameters": {
     "peeringName": "AzurePrivatePeering",
-    "crossConnectionName": "CrossConnection-6e918437-5077-4ffe-b45f-0b7c0672da4b",
+    "crossConnectionName": "6e918437-5077-4ffe-b45f-0b7c0672da4b",
```

Comment:
Please rename these as something generic like "connectionString", anything resembling actual keys should be avoided here

Reply:
Done